Constants.java

```diff
@@ -16,9 +16,20 @@ public final class Constants {
   public static class OperatorConstants {
     public static final int kDriverControllerPort = 0;
   }
-  public static class TalonFXConstants {
-    public static final int TALONFX_MOTOR_ONE_ID = 0;
-    public static final int  TALONFX_MOTOR_TWO_ID= 0;
+  public static class Subsystem{
+      public static class Talonfx  {
```

Comment:
use a more meaningfull  name for the class for example its inside subsystems so it name should be said subsystem